src/Components/SlateEditor/SlateEditor.js

```diff
@@ -19,6 +27,40 @@ export default function SlateEditor() {
     ]
   );
   const LIST_TYPES = ["numbered-list", "bulleted-list"];
+  const [showMenu, setShowMenu] = useState(false);
+  const [coordinates, setCoordinates] = useState("");
+  const allowedTags = [
+    {
+      label: "h1",
```

Comment:
are the user-visible labels same as the ones that appear on Notion? If not, make them similar